### PR TITLE
The tooltip label should have multiple entries in a new line

### PR DESCRIPTION
**Pull request by sudhirverma**
Fix: #219

### CODE REVIEW

File: src/functions/function-tree-view/functionsTreeItem.ts [hunk 1]
```diff
@@ -126,7 +126,7 @@ export class FunctionNodeImpl implements FunctionNode {
       this.contextValue === FunctionContextType.NOTCONNECTEDLOCALFUNCTIONS
     ) {
       return format(
-        `Name: ${this.CONTEXT_DATA[this.contextValue].tooltip} RunTime: ${this.runtime} Context: ${this.contextPath.fsPath}`,
+        `Name: ${this.CONTEXT_DATA[this.contextValue].tooltip}\nRuntime: ${this.runtime}\nContext: ${this.contextPath.fsPath}`,
```
Comment (mohitsuman) [inline src/functions/function-tree-view/functionsTreeItem.ts:129]: @sudhirverma is runtime the right word, or Language ? As in the UI we ask for `Select Language` ?

**Review by mohitsuman — CHANGES_REQUESTED** [review]

File: src/functions/function-tree-view/functionsTreeItem.ts (hunk 1 above)
Comment (sudhirverma) [reply src/functions/function-tree-view/functionsTreeItem.ts:129]: In func.yaml it displays `runtime: node` so I show runtime in tooltip.

Screenshot 2022-04-11 at 11 10 32 AM

Comment (sudhirverma) [reply src/functions/function-tree-view/functionsTreeItem.ts:129]: and in func list, it shows runtime.

```
func list -o json
[{"name":"test-demo-1","namespace":"default","runtime":"node","url":"http://test-demo-1.default.example.com","ready":"True"}]
```

Comment (mohitsuman) [reply src/functions/function-tree-view/functionsTreeItem.ts:129]: okay then it works.

**Review by mohitsuman — APPROVED** [review]

**Pull request merged**